### PR TITLE
Add command to show and notify Test Day helpers

### PR DESCRIPTION
**Pull request by galgeek**
hi @whimboo !

updates per issue #16

### CODE REVIEW

File: bot.js [hunk 1]
```diff
@@ -39,6 +40,8 @@ var ircServer = config.server,
     };
 
 function resetData() {
+  testDayAdmins = config.testDayAdmins;
```
Comment (whimboo) [inline bot.js]: While we are on this line, please name the config attribute `admins` so we are in sync with `helpers`.

File: bot.js [hunk 2]
```diff
@@ -104,6 +107,21 @@ client.addListener('message', function(from, to, message){
       client.say(to, "No etherpad is set.");
     }
   }
+  if (message.search('[!:]helpers') === 0){
+    if (testDay) {
+      client.say(to, "Today's helpers: " + helpers.join([separator = ', ']));
```
Comment (whimboo) [inline bot.js]: Why not simply `helpers.toString()`. Also `,` should be the default separator even in nodejs, or isn't it?

File: bot.js [hunk 3]
```diff
@@ -104,6 +107,21 @@ client.addListener('message', function(from, to, message){
...
+      if (message.indexOf('request') === 9) {
```
Comment (whimboo) [inline bot.js]: Best to split the message by ` ` and do a switch like `switch (arguments[1]) .. case 'request': set(); ... default: show()`

File: bot.js [hunk 4]
```diff
@@ -104,6 +107,21 @@ client.addListener('message', function(from, to, message){
...
+        helpersLength = helpers.length;
+        for (var i = 0; i < helpersLength; i++){
+          client.say(helpers[i], from + " could use some help!");
+        }
```
Comment (whimboo) [inline bot.js]: I don't see what was wrong before your last commit. Can you please explain? This adds more complexity.

File: bot.js (hunk 2 above)
Comment (galgeek) [inline bot.js]: The default separator is just `,` so `.toString` output is not very readable.

File: bot.js (hunk 4 above)
Comment (galgeek) [inline bot.js]: `helpers` is an array, not an object.
I'm rewriting this to use `forEach()`.

**Comment by galgeek** [comment]
hi @whimboo

thank you for your feedback! I've updated the PR (except as I've commented in the line notes above).

File: bot.js (hunk 2 above)
Comment (whimboo) [inline bot.js]: I see. But then helpers.join(', ') should also be enough.

File: bot.js [hunk 5]
```diff
@@ -12,7 +12,7 @@ var ircServer = config.server,
     client = new irc.Client(ircServer, nick, options),
     etherpad = "",
     testDay = false,
-    testDayAdmins = config.testDayAdmins,
+    testDayAdmins = config.admins,
```
Comment (whimboo) [inline bot.js]: please also fix the variable name.

**Comment by galgeek** [comment]
hi @whimboo 

the former testDayAdmins are now simply admins

thank you again for all your feedback!

**Comment by whimboo** [comment]
PR has been squashed and merged as https://github.com/mozilla/testdaybot/commit/931814e17c7b281e12aae4b85be869a34934efed. Thanks for your work!

**Pull request closed without merging**